proto/decentraland/kernel/apis/players.proto

```diff
@@ -23,7 +23,10 @@ message GetPlayersInSceneRequest {}
 message GetConnectedPlayersRequest {}
 
 service PlayersService {
+    // @deprecated, only available for SDK6 compatibility
```

Comment:
// Returns data about a specific player, by id
// NOTE: To be deprecated after implementing foreign-entities and once the avatar scene uses SDK7